Cargo.lock

```diff
@@ -5445,7 +5433,7 @@ dependencies = [
  "bytes",
  "libc",
  "memchr",
- "mio",
+ "mio 0.7.14",
```

Comment:
Hm, duplicating `mio` seems pretty bad, it's a core crate. Can we dedupe this as well?

Reply:
Done